test/unit/gateways/stripe_payment_intents_test.rb

```diff
@@ -351,6 +351,22 @@ def test_stored_credentials_does_not_override_ntid_field
     end.respond_with(successful_create_intent_response)
   end
 
+  def test_successful_off_session_purchase_with_claim_without_transaction_id
```

Comment:
should we also test for invalid, falsy ( not present/ `false`) values?

Reply:
Changes pushed up! 1c331bf